.github/workflows/CI.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.6]
+        python-version: [3.6, 3.7, 3.8, 3.x]
```

Comment:
Now I wonder if we should include 3.6 as it reached end of life in Dec '21 -  https://endoflife.date/python

Reply:
I think we can continue to support for sometime, then discontinue support.